src/scales/scale.time.js

```diff
@@ -176,6 +176,19 @@ module.exports = function(Chart) {
 			Chart.Scale.prototype.initialize.call(this);
 		},
 
+        /**
+         * Allow x-axis to be accessed via x or t attribute
+         */
+		getRightValue: function(rawValue) {
+			if (typeof(rawValue) === 'object') {
+				if ((rawValue instanceof Date) || rawValue.isValid) {
```

Comment:
Hmm. We do this same check in core.scale.js. I'm not sure I want to change it to `rawValue instance moment` because then we'd be introducing a dependency on moment in core